Source/Layout/ASLayout.mm

```diff
@@ -76,6 +76,13 @@ @implementation ASLayout
 
 @dynamic frame, type;
 
+static BOOL static_retainsSublayoutLayoutElements = NO;
```

Comment:
Any reason to have the prepended static here? Seems like a newish convention? Is there an existing one we can use?